docs/build/apps/dapp-frontend.mdx

```diff
@@ -247,34 +282,38 @@ Now let's add a new component to the `src/components` directory called `ConnectF
 </style>
 
 <script>
-  import { isAllowed, setAllowed, getUserInfo } from '@stellar/freighter-api';
+  import { kit, setPublicKey } from '../stellar-wallets-kit';
```

Comment:
I'm not well-versed in Astro, so this might be a stupid question. In SvelteKit, there's a "shortcut" for the `/src/lib/` directory (`$lib`), in Docusaurus (react-based) there's one for the project root (`@site/`), i think Vue has one for the project root, too (`@` maybe?).

Is there a similar kind of thing for Astro? Or, are relative file paths what's typically used?

Reply:
@ElliotFriend to be honest this was the first time I used Astro. I used direct file paths because it was the way the Stellar guides use them and that makes me think it's not something Astro supports

Reply:
Ah. Gotcha. Sounds solid enough to me hehe